ui/imports/shared/popups/walletconnect/DAppRequestModal.qml

```diff
@@ -0,0 +1,286 @@
+import QtQuick 2.15
+import QtQuick.Controls 2.15
+import QtQuick.Layouts 1.15
+import QtQml.Models 2.14
+import SortFilterProxyModel 0.2
+
+import QtGraphicalEffects 1.15
+
+import StatusQ 0.1
+import StatusQ.Core 0.1
+import StatusQ.Popups.Dialog 0.1
+import StatusQ.Controls 0.1
+import StatusQ.Components 0.1
+import StatusQ.Core.Theme 0.1
+
+import utils 1.0
+
+StatusDialog {
+    id: root
+
+    width: 480
+    implicitHeight: 680
+
+    required property string dappName
+    required property string dappUrl
+    required property url dappIcon
+    required property string accountDisplay
+    required property string signContent
+    required property string maxFeesText
+    required property string estimatedTimeText
+
+    signal sign()
+    signal reject()
+
+    function openWith() {
+        root.open()
+    }
+
+    closePolicy: Popup.CloseOnEscape | Popup.CloseOnPressOutside
+
+    title: qsTr("Sign request")
+
+    padding: 20
+
+    contentItem: ColumnLayout {
+        spacing: 20
+        clip: true
+
+        IntentionPanel {
+            Layout.fillWidth: true
+
+            dappName: root.dappName
+            dappIcon: root.dappIcon
+            accountDisplay: root.accountDisplay
+            signContent: root.signContent
+        }
+
+        ContentPanel {
+            Layout.fillHeight: true
+            Layout.fillWidth: true
+        }
+    }
+
+    header: StatusDialogHeader {
+        leftComponent: Item {
+            width: 46
+            height: 46
+
+            StatusSmartIdenticon {
+                anchors.fill: parent
+                anchors.margins: 3
+
+                asset: StatusAssetSettings {
+                    width: 40
+                    height: 40
+                    bgRadius: bgWidth / 2
+                    imgIsIdenticon: false
+                    isImage: true
+                    useAcronymForLetterIdenticon: false
+                    name: root.dappIcon
+                }
+                bridgeBadge.visible: true
+                bridgeBadge.width: 16
+                bridgeBadge.height: 16
+                bridgeBadge.image.source: "assets/sign.svg"
+                bridgeBadge.border.width: 3
+                bridgeBadge.border.color: "transparent"
+                bridgeBadge.color: "#2946C4"
```

Comment:
Hardcoded color? :)

Reply:
I can't seems to easily find this one. Do you have any hints? Or how should I add it.

Reply:
I used `Theme.palette.miscColor1` in the end and it seems to work fine.